.travis.yml

```diff
@@ -1,50 +1,101 @@
-  - secure: dQfZfrC73Oyj+6jEpCpMWIIUsr6Wm7WBLREw1X+nSxA3/ZkIqZ25xIs4kN/o4gv2ay43hwLpDKfeP4dnpSYt7mPu5T2wlP+5Ih1zrZFXJrUD5RkuRzeKHG3S4aKEJcLK85dYCdKLhtJXeaBOeOUTrUcrEXPlbTTW91uWU5ewuTsjmI0TDPesi5YvCB2V0tihyMBmnXzxwZHKnJYVfdGQhPKyMB5JetIW7prvQoJtsK4ViIQJjunlY9wB9ENu92mXYQ8SbMo1gCjT3+XDcrYRgTh56uKx9hzp1o1RvColW2+kWM3OEWScU2Z/GPzDvn37i3e6DVP3SOhS/mAW3I8FyeRpGSRlPbB2RJ+qyc9gBWYgh2L3Y5sNVi7UrOYOjOAA/KWa8XYqVg+iTYjpnzIt1Qj7NdhjVuDpe4MJhz20wNXPyffqsNSrRrc524GebymyKK1Yll1nHDQNfpOaHoWudt4s8AIAkMJFPcGOi5WWZvaW5jQWJrTuX1cMvtx4nX85jhm8piEC0PVhxoB3GaxVWktrB6QVXFueFM7YYhu8PZxOshOFVmPcwD6E3hmspuNB2IIbdBPYlYrLl07hOq+I/+J94Hi+8asnPvujFmKH0lnJZiC0pwfzLAXV+E1b3DcxDw7Uf/x/8tKa0LOVYLYYbI3PRpBvTTrCNKXSq/0asTQ=
+  - stage: format-check
+    name: Check with Scalafmt
+    script: sbt ++$TRAVIS_SCALA_VERSION fmtCheck
+    jdk: openjdk8
+    scala: 2.13.2
 
-before_install:
-- if [[ -n "${TRAVIS_TAG}" && "${TRAVIS_PULL_REQUEST}" = "false" ]]; then openssl aes-256-cbc -K $encrypted_d57b5eec735f_key
-  -iv $encrypted_d57b5eec735f_iv -in .travis/secrets.tar.enc -out .travis/local.secrets.tar
-  -d; tar xv -C .travis -f .travis/local.secrets.tar; fi
+  - stage: codecov-upload
+    name: Upload Codecov results
+    script: bash <(curl -s https://codecov.io/bash)
+    jdk: openjdk8
+    scala: 2.13.2
 
-before_script:
-- docker run -p 6379:6379 -d redis:alpine
-- docker run -p 6380:6379 -d eqalpha/keydb
+  - stage: release-212
+    name: Release for Scala 2.12
+    script: sbt ++$TRAVIS_SCALA_VERSION releaseIt
+    jdk: openjdk8
+    scala: 2.13.2
 
-script:
-- sbt -Dfile.encoding=UTF8 -Dsbt.color=always -Dsbt.supershell=false ++$TRAVIS_SCALA_VERSION fullBuild
+  - stage: release-213
+    name: Release for Scala 2.13
+    script: sbt ++$TRAVIS_SCALA_VERSION releaseIt
+    jdk: openjdk8
+    scala: 2.13.2
 
-before_cache:
-- find $HOME/.sbt -name "*.lock" | xargs rm
-- find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
+  - stage: cleanup
+    name: Docker shutdown
+    script: docker stop $(docker ps -a -q)
+    jdk: openjdk8
+    scala: 2.13.2
 
-after_success:
-- if [[ -n "${TRAVIS_TAG}" && "${TRAVIS_PULL_REQUEST}" = "false" ]]; then sbt -Dsbt.color=always -Dsbt.supershell=false ++$TRAVIS_SCALA_VERSION releaseEarly;
-  fi
-- bash <(curl -s https://codecov.io/bash)
+services:
+- docker
 
-after_script:
-- docker stop $(docker ps -a -q)
+cache:
+  directories:
+  - "$HOME/.coursier"
```

Comment:
The cache dir is different now see https://get-coursier.io/docs/cache.html#default-location